devel/unibilium/files/patch-Makefile.diff

```diff
@@ -1,11 +1,14 @@
-diff --git a/Makefile b/Makefile
---- a/Makefile
-+++ b/Makefile
-@@ -33,7 +33,7 @@
+--- a/Makefile.orig	2018-02-08 16:14:10.000000000 +0100
++++ b/Makefile	2018-09-14 22:39:48.000000000 +0200
+@@ -33,11 +33,7 @@
  MAN3DIR=$(MANDIR)/man3
  
  ifneq ($(OS),Windows_NT)
--  TERMINFO_DIRS="/etc/terminfo:/lib/terminfo:/usr/share/terminfo:/usr/lib/terminfo:/usr/local/share/terminfo:/usr/local/lib/terminfo"
+-  TERMINFO_DIRS="$(shell ncursesw6-config --terminfo-dirs 2>/dev/null || \
+-                         ncurses6-config  --terminfo-dirs 2>/dev/null || \
+-                         ncursesw5-config --terminfo-dirs 2>/dev/null || \
+-                         ncurses5-config  --terminfo-dirs 2>/dev/null || \
+-                         echo "/etc/terminfo:/lib/terminfo:/usr/share/terminfo:/usr/lib/terminfo:/usr/local/share/terminfo:/usr/local/lib/terminfo")"
 +  TERMINFO_DIRS="/etc/terminfo:$(PREFIX)/share/terminfo:/usr/local/share/terminfo:/usr/share/terminfo"
```

Comment:
As `/etc/terminfo` does exist anyway on macOS, the new default of using `ncursesw6-config --terminfo-dirs` to determine the path looks fine to me.

Reply:
Thanks for reviewing, @raimue!

Reply:
The problem with that approach, as I see it, is the following:
- this adds an unnecessary dependency on `ncurses`, or a non-reproducible build otherwise
- if `ncurses` port is installed, it returns *only* `${PREFIX}/share/terminfo`

I deliberately left the hardcoded list because:
- `/etc/terminfo` is for administrators to be able to locally override terminfo entries, if they choose to do so
- `${PREFIX}/share/terminfo` is from the `ncurses` port, which will be reasonably up-to-date
- `/usr/local/share/terminfo` is either from Homebrew or a local install of ncurses, which will tend to be more up-to-date then the system version
- `/usr/share/terminfo` the system version, if everything else fails

By *not* using `ncurses-config` we still allow all these paths to be recognized simultaneously, leaving maximum flexibility to the users. I don't think we should take that away —  it would be a regression, in my opinion.

Reply:
Okay, yes, that dependency should not be necessary. Thanks for the detailed explanation.